css/content.css

```diff
@@ -4,7 +4,7 @@
     flex:1 1 0%;
 }
 
-.outlay{
+.outlay .alert-box{
     border-radius: 4px;
     min-height: 100px;
```

Comment:
use rem unit

Reply:
Instead of small decimal notation i thought it would be better to use px ..example 5px is 0.25 rem to avoid such calculations i did like that

Reply:
updated